SConstruct

```diff
@@ -238,6 +238,28 @@ preSfm = command("sfm", [skymap, transmissionCurvesTarget],
 env.Depends(preSfm, refcat)
 sfm = {(data.visit, data.ccd): data.sfm(env) for data in sum(allData.values(), [])}
 
+# Sky correction
+def processSkyCorr(dataList):
+    """Generate sky corrections"""
+    visitDataLists = defaultdict(list)
+    for ff in allData:
+        for data in allData[ff]:
+            visitDataLists[data.visit].append(data)
```

Comment:
Interesting idea to use `defaultdict`; for this pattern I always use a regular dict with `d.setdefault(key, []).append(...)`, but I think yours might be clearer.